drivers/lcd/ssd1680.c

```diff
@@ -1011,21 +1021,35 @@ static void ssd1680_snd_cmd_with_data_bitstrip(
     {
       for (j = 0; j < 8; j++)
         {
+#    if defined(CONFIG_LCD_LANDSCAPE)
+          if (offset == 0)
+            {
+              bytes[j] = *(dta + j * strip_len);
+            }
+          else
+            {
+              bytes[j] = (*(dta + j * strip_len)) >> offset;
+
+              bytes[j] |= ((*(dta + j * strip_len - 1)) << (8 - offset)
+                           & (0xff << (8 - offset)));
+            }
+#    else
           bytes[j] = *(dta + j * strip_len);
+#    endif
         }
 
       for (i = 0; i < 8; i++)
         {
           val = 0;
           for (j = 0; j < 8; j++)
             {
-#if defined(CONFIG_LCD_LANDSCAPE)
+#    if defined(CONFIG_LCD_LANDSCAPE)
               val |= ((bytes[j] << (7 - j)) & (1 << (7 - j)));
               bytes[j] = bytes[j] >> 1;
-#elif
+#    elif
```

Comment:
```suggestion
#    else
```

Reply:
Note, still not fixed here and in a few places below.

Reply:
Shame on me. This bug was in many places. In first implementation it was considered PORTRAIT RPORTRAIT, LANDSCAPE and RLANDSCAPE.... That's why instead of else there was elif. It was forgotten to change elif into else after removing condition. Thanks a lot. 
I checked whole file.